include/irr/asset/CTexturePacker.h

```diff
@@ -58,40 +58,46 @@ class CSquareAddressAllocator
 class ITexturePacker
 {
 protected:
-    _IRR_STATIC_INLINE_CONSTEXPR uint32_t ADDR_LAYER_SHIFT = 10u;//10bit 2d page address ( 32=2^(10/2) pages per dimension ) and up to 6bit for 3rd address dimension (layer)
-    _IRR_STATIC_INLINE_CONSTEXPR uint32_t ADDR_MORTON_MASK = (1u<<ADDR_LAYER_SHIFT)-1u;
+    const uint32_t ADDR_LAYER_SHIFT;
+    const uint32_t ADDR_MORTON_MASK;
+    const uint32_t TEX_OFFSET_X_MASK;
 
     const uint32_t m_pgSzxy;
     const uint32_t m_tilesPerDim;
     CSquareAddressAllocator<uint32_t> m_pgtAddrAlctr;
 
 public:
+    using page_tab_offset_t = core::vector2du32_SIMD;
+    static page_tab_offset_t page_tab_offset_invalid() { return page_tab_offset_t(~0u,~0u,~0u); }
```

Comment:
why are you now returning a 128bit value instead of a 32bit?

Reply:
because why not, bitdepth doesnt matter as long as its sufficient for all values

Reply:
yeah, but when they get copied over make sure shaders use 64bit as input/storage